### PR TITLE
Sane Shotgun Rebalance

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request
An edit of #426 that keeps non-lethal rounds a bit higher up. Meaning:

- Beanbag slugs go from 55 -> 50 stamina damage.

- Buckshot pellets from 12.5 -> 8, totaling 48 damage point-blank

- Stamina damage falloff goes from 0.5 -> 0.6

All other changes match the original PR

## Why It's Good For The Game
Most of sparrows changes were good, but nerfing non-lethals seems redundant.

## Changelog
:cl:
balance: shotguns are generally worse.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser/wall
 
 /obj/projectile/beam/weak
-	damage = 15
+	damage = 6
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/beams.dm:40]:
```suggestion
	damage = 6		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 2]
```diff
@@ -1,11 +1,12 @@
 /obj/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
-	damage = 60
+	damage = 30
+	armour_penetration = 10
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:4]:
```suggestion
	damage = 30		// Wasp Edit - Shotgun Nerf
	armour_penetration = 10		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 
 /obj/projectile/bullet/shotgun_beanbag
 	name = "beanbag slug"
 	damage = 5
-	stamina = 55
+	stamina = 50
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:9]:
```suggestion
	stamina = 50		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 4]
```diff
@@ -18,18 +19,19 @@
 /obj/projectile/bullet/shotgun_stunslug
 	name = "stunslug"
 	damage = 5
-	paralyze = 100
+	stamina = 40
+	knockdown = 10
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:23]:
```suggestion
	stamina = 40		// Wasp Edit - Shotgun Nerf
	knockdown = 10		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 5]
```diff
@@ -18,18 +19,19 @@
...
 	stutter = 5
 	jitter = 20
-	range = 7
+	range = 4
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:26]:
```suggestion
	range = 4		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 6]
```diff
@@ -18,18 +19,19 @@
...
 	icon_state = "spark"
 	color = "#FFFF00"
 
 /obj/projectile/bullet/shotgun_meteorslug
 	name = "meteorslug"
 	icon = 'icons/obj/meteor.dmi'
 	icon_state = "dust"
-	damage = 30
+	damage = 25
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:34]:
```suggestion
	damage = 25		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 7]
```diff
@@ -56,17 +58,17 @@
 	return BULLET_ACT_HIT
 
 /obj/projectile/bullet/pellet
-	var/tile_dropoff = 0.75
-	var/tile_dropoff_s = 0.5
+	var/tile_dropoff = 0.8
+	var/tile_dropoff_s = 0.6
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:62]:
```suggestion
	var/tile_dropoff = 0.8		// Wasp Edit - Shotgun Nerf
	var/tile_dropoff_s = 0.6		// Wasp Edit - Shotgun Nerf
```

Could also probably be put in the waspstation folder

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 8]
```diff
@@ -56,17 +58,17 @@
...
 
 /obj/projectile/bullet/pellet/shotgun_buckshot
 	name = "buckshot pellet"
-	damage = 12.5
+	damage = 8
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:66]:
```suggestion
	damage = 8		// Wasp Edit - Shotgun Nerf
```

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 9]
```diff
@@ -56,17 +58,17 @@
...
 
 /obj/projectile/bullet/pellet/shotgun_rubbershot
 	name = "rubbershot pellet"
-	damage = 3
-	stamina = 11
+	damage = 1
+	stamina = 8
```
Comment (Superyodeler) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:71]:
```suggestion
	damage = 1		// Wasp Edit - Shotgun Nerf
	stamina = 8		// Wasp Edit - Shotgun Nerf
```

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Needs margularizing

**Comment by AsciiSquid** [comment]
Closing because this PR is bad and I am bad for making it

**Pull request closed without merging**